.github/workflows/ssgts-cs8.yaml

```diff
@@ -0,0 +1,178 @@
+name: SSGTS CS8
+on:
+  pull_request:
+    branches: [ master, 'stabilization*' ]
+jobs:
+  build-content:
+    name: Build Content
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:latest
```

Comment:
This is no longer needed in CentOS jobs right? If so we should remove it to save time pulling the image from the registry.

Reply:
We still need fedora here as this is the step we build the content. Otherwise it would build the content on ubuntu with an outdated openscap. The place where the image is pulled for SSGTS is in a different place.